217.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @param {number[]} nums
+ * @return {boolean}
+ */
+var containsDuplicate = function (nums) {
+  nums.sort((a, b) => a - b);
+
+  for (let i = 0; i < nums.length; i++) {
```

Comment:
You can reduce the number of loops by 1 because the last loop is unnecessary.
```suggestion
  for (let i = 0; i < nums.length - 1; i++) {
```